core/src/main/scala/chisel3/internal/BiConnect.scala

```diff
@@ -251,20 +251,22 @@ private[chisel3] object BiConnect {
     }
   }
 
-  /** Check whether two aggregates can be bulk connected (<=) in FIRRTL. From the
-    * FIRRTL specification, the following must hold for bulk connection:
+  /** Check whether two Data can be bulk connected (<=) in FIRRTL. From the
+    * FIRRTL specification, the following must hold for FIRRTL connection:
     *
-    *   1. The types of the left-hand and right-hand side expressions must be
+    *   1. The Chisel types of the left-hand and right-hand side expressions must be
```

Comment:
why would the firrtl specification say anything about the Chisel types?

Reply:
It doesn't, but in order for us to know if a FIRRTL connection can be emitted, we need to know whether Chisel type is a valid FIRRTL type in this regard

Reply:
I clarified, thanks!